CMake/Dependencies/libkvscproducer-CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ include(ExternalProject)
 # clone repo only
 ExternalProject_Add(libkvscproducer-download
 	GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-producer-c.git
-	GIT_TAG           cf21f56d5bc47d9a64d983438fcc39fbefa352b7
+	GIT_TAG           configure-timeouts
```

Comment:
To update to using commit once producer C PR is merged: https://github.com/awslabs/amazon-kinesis-video-streams-producer-c/pull/366